daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -1157,14 +1157,14 @@ final case class Compiler(packages: PackageId PartialFunction Package) {
       // actors are either the singleton set of submitter of an exercise command,
       // or the acting parties of an exercise node
       // of a transaction under reconstruction for validation
-      actors: SExpr): SExpr = {
```

Comment:
explain why this is optional now

Reply:
That's a good catch. It's actually never `Some` any more, so I removed it as a parameter now. In general, exercise actors are computed from the controllers in the new versions of LF, and the compiler does not set them any more. I've added a comment to `compileExercise` to this effect.